src/mgr/DaemonServer.cc

```diff
@@ -78,15 +78,15 @@ DaemonServer::DaemonServer(MonClient *monc_,
 					g_conf().get_val<uint64_t>("mgr_client_messages"))),
       osd_byte_throttler(new Throttle(g_ceph_context, "mgr_osd_bytes",
 				      g_conf().get_val<Option::size_t>("mgr_osd_bytes"))),
-      osd_msg_throttler(new Throttle(g_ceph_context, "mgr_osd_messsages",
+      osd_msg_throttler(new Throttle(g_ceph_context, "mgr_osd_messages",
```

Comment:
Ugh.  After some digging, I think these three will not break Ceph, in fact they may address a latent bug.

That said, I *think* these three will change perfcounter output, arguably to fix it, so my suggestion here is to accept this change, and to add a blurb to the release notes if indeed the perfcounter output changes.
@zdover23 @epuertat your thoughts?

Reply:
> I _think_ these three will change perfcounter output

This is correct, a release note would be needed.  Thanks for digging!

Reply:
BTW, @nizamial09 @aaSharma14 @pereman2: I'm almost sure we aren't, but we should double check whether some of these perf-counters are used in Grafana/AlertManager, since that might require an update there too.

Reply:
I did a find | grep of `ceph/ceph`, were they used as above wouldn't they be in there?

Reply:
Yeah, just wanted to confirm. Since @jsoref broke the original PR down to several ones based on CODEOWNERS map, those changes would show up in the Dashboard PR.

Reply:
Can someone explain what I need to do for this?

I half expect to leave this commit out of my squash, but I also expect to need to do something for the release note side and I haven't done one of those yet.

Reply:
I think release notes would be up to us (well maybe Ernesto and Ilya) not you. They get assembled by the core team.

Reply:
Anyone can add a release note -- it's just a entry in `PendingReleaseNotes` file in the respective release section.  It's best to add release notes in the same commit/PR that is bringing the change.

Release notes are "assembled" only when the release is cut, they aren't public until then.  In this case, my understanding is that this PR is not getting backported, so it's targeted at the S (19.y.z) release.  We don't have a section for it yet, so you would need to open it:
```
>=19.0.0
* <release note text>

```
at the top of the file.

Reply:
@jsoref ^